index.html

```diff
@@ -8,6 +8,15 @@
     <title>Braintech | IT Solutions and Technology Startup HTML Template</title>
```

Comment:
Not about favicons :). In the title shouldn't we remove the text "HTML Template"?

Reply:
yes, original has it, but we should change it